### ISSUE TITLE
Install failed

### ISSUE THREAD

**Issue by jcrocks2000** [Jays2Kings/DS4Windows#70]
Whenever I open up DS4 windows and try to download the driver, it just says opening installer for like 3 minutes and then says install failed, please retry. I have retried multiple times and still nothing. Please help.

**Comment by westonbelk** [comment]
Make sure to install Microsoft .NET 4.5 or higher as mentioned in the readme. Doing this fixed it for me.

**Comment by Hubris4** [comment]
Framenetwork 4.5 or higher has already been installed. still cant get it to install

**Comment by garybonds** [comment]
I have the identical same problem. Verified with .NET Framework Setup Verification Utility that I have .NET Framework
4.6, 4.6.1, 4.5, 4.5.1 and 4.5.2

Running on fairly recent Windows 10 (never upgraded, clean install)

**Comment by garybonds** [comment]
Seems to be an issue with having to install 
Microsoft Visual C++ 2010 Redistributable Package
Microsoft Visual C++ 2013 Runtime

I downloaded latest version of ScpToolkit and let it run its prerquisites check (without installing ScpToolkit). After that the driver downloaded and installed without problem.

**Comment by Hubris4** [comment]
Turns out all i had to do was restart my computer

**Comment by Rose2234** [comment]
yeah this project is and he does not care anymore i rather give my money to a developer that ACTUALLY cares about his work and not give out cliche excuses like "oh i have been way too busy". and his site does not work anymore it says Bandwidth has been exceeded the limit. Xpadder here i come

**Comment by Kryptonit3-zz** [comment]
Install this app

https://github.com/nefarius/ScpToolkit#installation-how-to

After install, at end you will see a green button that says "Install Drivers", run it. Once done, restart DS4Windows, controller should show up once you connect via USB or Bluetooth.

**Comment by Rose2234** [comment]
yeah i can't ope that file windows is saying your innternet settings has prevented you from opening this file. i have tried everything to try to fix it but you cannot fix. tried the cliche fix like your internet settings and disable it.

**Comment by rhx3030** [comment]
Kryptonit3 // that's great solution !!
                     very very very thankyou !!!

**Comment by rhx3030** [comment]
go to https://github.com/Ryochan7/DS4Windows/releases
it's a great solution

**Comment by Aerobbo** [comment]
Kryptonit3, Thanks, your way helped me a lot! Again Thx dude you saved my life ;)

**Comment by Raxvis** [comment]
[Virtual Bus Driver.zip](https://github.com/Jays2Kings/DS4Windows/files/2221480/Virtual.Bus.Driver.zip)

Here is the driver if anyone still needs it
